docs/guide/index.md

```diff
@@ -117,7 +117,7 @@ In a project where Vite is installed, you can use the `vite` binary in your npm
   "scripts": {
     "dev": "vite", // start dev server
     "build": "vite build", // build for production
-    "serve": "vite preview" // locally preview production build
+    "preview": "vite preview" // locally preview production build
```

Comment:
If we embed json with comments in markdown, we should always use `jsonc` instead of `json`
Then the comments will be rendered correctly

Reply:
Interesting, let's change that in another PR for all examples that have comments?